client/src/common/components/Dropdown.jsx

```diff
@@ -32,25 +45,26 @@ class Dropdown extends PureComponent {
     document.removeEventListener('click', this.handleClickOutside);
   }
 
+  hideDropdown = () => this.setState({ isVisible: false });
+
+  showDropdown = () => this.setState({ isVisible: true });
```

Comment:
This is a Dropdown component, IMHO this should be simplified to `show/hide` as there is nothing else than the dropdown to be shown/hidden.